screens/authentication/auth.css

```diff
@@ -0,0 +1,43 @@
+/* Container */
+.card {
+  flex-direction: column;
+}
+
+/* Button */
+.btn-link-secondary {
+  color: var(--secondary-color);
+  padding: 0;
+}
+
+.btn-link-secondary:hover {
+  color: var(--secondary-color-light);
+}
+
+.info-container {
+  justify-content: space-between;
+}
+
+.checkbox {
+  width: 1.5rem;
+  height: 1.5rem;
+}
+
+.bottom-nav-container {
+  display: flex;
+  align-items: center;
+}
+
+.goto-icon {
+  font-size: 2rem;
+}
+
+.toggle-icon-container {
+  position: relative;
+}
+
+.toggle-icon {
+  position: absolute;
+  top: 1rem;
+  right: 1rem;
```

Comment:
better to use percentage for absolute positions

Reply:
> better to use percentage for absolute positions

Changes made, Thanks for the feedback.